docs/en/get_started.md

```diff
@@ -89,14 +91,16 @@ Option (b). If you install mmdetection with pip, open you python interpreter and
 
 ```python
 from mmdet.apis import init_detector, inference_detector
+from mmdet.utils import register_all_modules
 
+register_all_modules()
 config_file = 'yolov3_mobilenetv2_320_300e_coco.py'
 checkpoint_file = 'yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth'
 model = init_detector(config_file, checkpoint_file, device='cpu')  # or device='cuda:0'
-inference_detector(model, 'demo/cat.jpg')
+inference_detector(model, 'demo/demo.jpg')
 ```
 
-You will see a list of arrays printed, indicating the detected bounding boxes.
+You will see a result of `DetDataSample`, indicating the detected bounding boxes, labels, and scores.
```

Comment:
It should be a list of DetDataSample. And tell users that the predictions are in the `pred_instance` field.